hw03_Sugargod1.py

```diff
@@ -0,0 +1,130 @@
+
+# TODO Do not edit this file directly! Instead, create a new file called
+#        hw03_username.py and copy this code into it!
+
+#################################################################################
+# Author: Sugarragchaa Tumur-Ochir
+# Username: Sugargod1
+#
+# Assignment: hw03_Sugargod1
+# Purpose: Better understanding of git and Github
+# Google Doc Link: https://docs.google.com/document/d/17NHKwPKQIHpA5JfAf-XqfP_sFllP3Gk8PNOkkDWPsdk/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements:
+# screen function determines screen color
+# housebody function make house's body
+# roofy makes roof
+# doory makes the door
+# windowy makes the window
+# we only used one hardworking turtle, he does them all
+# I'm currently working on my datascience and algorithm skill's so I didn't show much creativity in it. UwU feeling guilty.
```

Comment:
The Acknowledgement section is for citing sources. So, if you searched for help in doing something, cite it here. If a friend helped you, tell me here... It is not intended for documentation (docstrings do that).